docs/api/topics/firefox_os_addons.rst

```diff
@@ -38,6 +38,7 @@ Detail
         {
           "id": 1,
           "description": null,
+          "disabled": false,
```

Comment:
We probably want to be consistent with other APIs, but I like booleans prefixed with `is_` or `has_`.